Exercise 23/index.js

```diff
@@ -0,0 +1,17 @@
+let currentDay = prompt("Thanks for contacting Customer Support. As this time, we are only able to provide customer support Monday - Friday, 8am-6pm, Please enter the current day of the week");
+let currentHour = prompt("Please enter current hour in 24hr format (values accepted would be a number between 0-23)");
+
+currentWeekday = currentDay.toUpperCase();
+currentHour = Number(currentHour);
+
+if (currentWeekday === "SATURDAY" || currentWeekday === "SUNDAY") {
+    alert("Our customer support is not available on the weekend. Feel free to contact us Monday-Friday 9-5pm. As a reminder Google is availabe 24-7.");
+} else {
+    if (currentHour < 0 || currentHour > 23) {
+        alert("We are unable to process your request as the time you have entered is not valid.");
+    } else if (currentHour > 18 || currentHour < 8) {
```

Comment:
Good job @Emullan04 
I don't think they'd like it if you expected them to work after 6pm on weekdays?

Reply:
Error corrected! Please review